frontend/app/work_packages/directives/work-package-column-directive.js

```diff
@@ -114,14 +114,18 @@ module.exports = function(PathHelper, WorkPackagesHelper, UserService){
       function displayDataAsLink(id) {
         // Example of how we can look to the provided meta data to format the column
         // This relies on the meta being sent from the server
-        scope.displayType = 'link';
-        scope.url = getLinkFor(id, scope.column.meta_data.link);
+        var link_meta = scope.column.meta_data.link;
```

Comment:
Identifier 'link_meta' is not in camel case.<br>Identifier 'meta_data' is not in camel case.